kairos-test-utils/src/cctl.rs

```diff
@@ -30,15 +30,20 @@ pub struct CasperNode {
 }
 
 pub struct CCTLNetwork {
-    pub working_dir: TempDir,
-    pub assets_dir: PathBuf,
```

Comment:
**Comment:** Assets directory path was used in `drop()`, and now we are basing on assumption it's placed directly in the workdir and named `assets`. That is okay though.